omnibus/config/software/system-probe.rb

```diff
@@ -17,7 +17,7 @@
   mkdir "#{install_dir}/embedded/share/system-probe/ebpf/co-re/btf"
   mkdir "#{install_dir}/embedded/share/system-probe/java"
 
-  copy 'pkg/network/java/agent-usm.jar', "#{install_dir}/embedded/share/system-probe/java/"
+  copy 'pkg/network/protocols/tls/java/agent-usm.jar', "#{install_dir}/embedded/share/system-probe/java/"
```

Comment:
please add to the QA steps, to ensure `agent-usm.jar` exists in the final docker artifact

Reply:
added to QA testing section